app/views/service_worker/index.js.erb

```diff
@@ -39,9 +39,15 @@
     return [
       '/%F0%9F%92%B8',         // 💸 (hiring)
       '/admin',                // Don't run on admin dashboard.
+      '/new',                  // Don't run on editor.
+      '/enter',                // Don't run on sign in page.
       '/oauth/',               // Skip oauth apps
       '/onboarding',           // Don't run on onboarding.
+      '/shop',                 // Don't run on Shop
+      '/sidekiq',              // Skip for Sidekiq dashboard
       '/users/auth',           // Don't run on authentication.
+      '/users/sign_in',        // Don't run on sign in page.
+      '/welcome',              // Don't run on welcome reroutes.
```

Comment:
I added some paths that I think have forms that could be problematic (see description). Maybe worth considering to add more to the list that @Zhao-Andy mentioned [here](https://github.com/forem/forem/pull/12953#discussion_r592480071). I don't think we need all of them but open to adding more paths to this list if others consider they should be skipped